### ISSUE TITLE
UI moves when clicked on

### ISSUE THREAD

**Issue by ItsRogueRen** [open-goal/launcher#581]
### Acknowledgements

- [x] I have [searched](https://github.com/open-goal/launcher/issues?q=is%3Aissue+is%3Aopen+label%3Abug+sort%3Aupdated-desc) for duplicate or closed bug reports
- [x] I understand that I am supposed to provide my own legitimately obtained copy of the game
- [x] If possible, I've attached the support package zip file to the issue [see how here](https://github.com/open-goal/launcher#asking-for-help)

### Describe the Bug

Minor UI issue with the Linux version, the buttons for Play, Features, Advanced, and options move when clicked on, making some menu items impossible to click without them randomly not moving one time.

### How To Reproduce

Install the app from the website on Linux

Open the appimage

Install a game from an ISO

Click a button other than play and then a menu item

### Operating System

Nobara 40/Fedora 40

### Launcher Version

2.5.0

### Tooling Version

0.2.17

**Comment by xTVaser** [comment]
Does this occur on the version that just released (2.5.1)?

**Comment by ItsRogueRen** [comment]
> Does this occur on the version that just released (2.5.1)?

Yes, but it didn't start until I clicked Advanced and then it started doing it like before

EDIT I tried to add an gif of it but github is rejecting the file
